pkg/network/infraconfigurators/masquerade.go

```diff
@@ -136,6 +136,10 @@ func (b *MasqueradePodNetworkConfigurator) PreparePodNetworkInterface() error {
 		return err
 	}
 	if ipv4Enabled {
+		err = b.handler.ConfigureRouteLocalNet(iptables.ProtocolIPv4, api.DefaultBridgeName)
+		if err != nil {
+			log.Log.Reason(err).Errorf("failed to configure routing of local addresses for %s", api.DefaultBridgeName)
```

Comment:
The code bellow returns if an error occurs. Can the new flow continue in case of an error?

Reply:
Thanks, fixed.